src/mpyl/project.py

```diff
@@ -631,6 +658,24 @@ def load_project(
             raise
 
 
+def merge_dicts(child_yaml_values, parent_yaml_values):
```

Comment:
same as above and don't forget to add some types here

Reply:
Addressed in https://github.com/Vandebron/mpyl/pull/287/commits/91bb7489d9cfdacc6f7855d6af138c8025478c99

Reply:
Hm, the naming is still wrong though? In 95% of the cases it's not a `child_yaml` but just a yaml right?

Reply:
https://github.com/Vandebron/mpyl/pull/287/commits/7e7a3c7ec43c20dfc33b926466c73d79d02f17ea